README.en.md

```diff
@@ -0,0 +1,89 @@
+# GOB
+
+Generic Disclosure of Basic Data
+
+_Een Nederlandse versie van dit document is beschikbaar in [README](README.md)_
+
+## Background
+
+Basic data is the raw material for a large number of municipal processes.
+Without these data, the processes can not be executed.
+
+Basic data are recorded in basic registers (prescribed by law) and core registrations (prescribed by the Municipal Executive).
+
+Some of these data are taken from their own municipal facilities (source system) and partly from national facilities.
+The problem is that all these sources do not use one and the same data / message standard.
+In-house customers make use of different systems in which the basic data are shown or processed.
+
+The seven basic registers are:
+1. persons (BRP)
+2. addresses and buildings (BAG)
+3. value of immovable property (BR WOZ)
+4. small-scale topography (BRT)
+5. large-scale topography (BGT)
+6. Commercial register (HR)
+7. Land Registry (BRK).
+
+The eight core registrations are:
+1. the municipal restriction registration (Wkpb)
+2. the current height file in the Netherlands
+3. areas
+4. NAP
+5. aerial photographs
+6. measuring bolts
+7. monuments
+8. panorama images.
+
+Three systems are in use for retrieving data from the source or the national facility, the management, the translation and the distribution of the basic data:
+- DIVA for real estate data
+- Makelaar suite for personal details
+- Handelsregister and Neuron Communicator for BAG and WOZ data
+
+Not all systems can handle the different deliveries.
+A 'translation' is therefore necessary.
+Within GOB, the basic data from the various source registrations are stored, processed and distributed in a format as defined in
```

Comment:
Gaat het hier over de vorm waarin werd gecommuniceerd zoals `StUF` of `UVA` of over de de inhoud van de berichten? Stelselpedia gaat namelijk maar beperkt in op de vorm, wel op de inhoud.

Reply:
Weliswaar een logisch gegevensmodel maar daar is wel een technisch datamodel van af te leiden dat is opgesteld volgens de naamgevingen in Stelselpedia en RSGB

Reply:
Probleem is alleen dat wanneer het logisch model leidend is voor wat de GOB implementeert je mogelijk tegen dezelfde problemen aanloopt die er nu ook met bijvoorbeeld de StUF implementatie zijn.

Reply:
Mee eens, het model is daarom ook als indicatief te zien, het is zeker niet op te vatten als beperkt tot. Maar je kunt ervan uitgaan dat wat in Stelselpedia staat ook onder een min of meer vergelijkbare naam te vinden zal zijn in GOB. En daarnaast nog wel meer data. Denk bijvoorbeeld aan CBS namen voor gebieden.
Daarnaast passen we versioning toe zodat we niet opgesloten worden in welk formaat dan ook maar vrijheid hebben om door te groeien.

Reply:
De kern is voornamelijk dat de oude StUF is gemaakt voor het synchroon houden van systemen waarbij niet wordt van wordt uitgegaan dat de ontvanger de historische continuiteit probeert te waarborgen. Dat kan met de StUF (impelementatie van Amstedam namelijk maar beperkt). Dat heeft dus wezenlijke invloed op hoe er `by-design` wordt nagedacht over de implementatie (en invulling) van berichten. Hetzelfde speelt er ook bij de BAG wat betreft de UVA2 leveringen. Die worden dan wel historisch aangeboden, maar missen informatie om een feilloze historisch consistente schaduw op te zetten. De vrees is dat wanneer deze aspecten dus niet vanaf de ontwerp tafel meegenomen worden we in een implementatie terecht komen met dezelfde beperkingen als nu.

Reply:
Goed dat je dit opmerkt. Historie is nu juist een integraal onderdeel van GOB. Feitelijk is de architectuur gebaseerd op een boekhouding, met journaalposten en rapportages. Door het vastleggen van alle mutaties als journaalposten is het model slechts een afgeleide geworden. Historisch snapshots, tussentijdse rapportages, het is allemaal mogelijk. En ook een undo, dat is namelijk een normale gang van zaken in een boekhouding, gewoon correctieposten maken. Mocht je toegang hebben tot docuwiki nodig ik je uit om kennis te nemen van de details van de architectuur en alle overwegingen die daaran ten grondslag liggen.

Reply:
Wow wat een requirements. Ik ben wel nieuwsgierig geworden. Een correcte feilloze huidige presentatie was/is al moeilijk genoeg!

Reply:
> dat is namelijk een normale gang van zaken in een boekhouding, gewoon correctieposten maken.

En het is goed dat die aanname een requirement is geworden! Ik hoop dat dat ook getoetst wordt door de partijen die er op die manier van afhankelijk zijn. Zoals onze eigen afdeling Basisstatistiek. Voorheen was die aanname er namelijk ook (zoals je zelf al zegt), maar is dat nooit tot een sluitende implementatie gekomen.

> Mocht je toegang hebben tot docuwiki nodig ik je uit om kennis te nemen van de details van de architectuur en alle overwegingen die aan de architectuur ten grondslag liggen.

Ga ik zeker doen. Dank.